commons/services/browser.py

```diff
@@ -179,21 +179,14 @@ def wait_for_element(self, by: By, value: str, condition="visible", timeout=None
                 f"Failed to wait for element {value} to become {condition}: {e}")
             raise
 
-    def execute_script(self, script, *args):
-        """
-        
-        Executes JavaScript in the context of the current page.
-        
-        :param script: The JavaScript code to execute.
-        :param args: Arguments to pass to the JavaScript code.
-        :return: The result of the executed script.
-        """
+    def execute_script(self):
+        #Executes JavaScript in the context of maher login page.
         try:
             if self.driver:
-                self.logger.info(f"Executing script: {script}")
-                result = self.driver.execute_script(script, *args)
-                self.logger.info(f"Script executed successfully, result: {result}")
+                self.logger.info("Executing script")
+                result = self.driver.execute_script("return window.sessionStorage.getItem('token');")
```

Comment:
no hard coding require, it should take the script we want to run as a parameter ( string )